src/CruxComponent.tsx

```diff
@@ -395,7 +395,7 @@ export class CruxComponentCreator {
                         </div>}
                         <div style={{ marginTop: "10px" }} />
 
-                        <Table className="table table-striped cftable" striped bordered condensed hover>
+                        <Table className="table table-striped cftable" striped bordered condensed hover responsive>
                             <thead>
```

Comment:
Added responsive attribute. which will add horizontal scroll for table it is sized exceeded screen width